Week-13/weather/services/weather.js

```diff
@@ -0,0 +1,96 @@
+const axios = require("axios");
+
+const { weatherAPIBaseUrl, weatherAPIKey } = require("../config");
+const { checkIsFutureDate } = require("../utils/utils");
+
+const getParamsAndQueryFromUrl = ({ query, params }) => {
+  const { cityName } = params;
+  const {
+    cityName: queryCityName,
+    cityCode,
+    pageNumber = 1,
+    days,
+    date,
+    hour,
+  } = query;
+
+  if (days && (+days > 10 || +days < 0)) {
```

Comment:
Can this be handled in the front end itself? We can restrict making an API call. Kindly share your thoughts.

Reply:
Yes, but we don't have UI right now, I think this is backend assignment only